### ISSUE TITLE
Bug tracker isn't configured to send all changes to an email list

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11496]
|[<img alt="jyknight's avatar" src="https://avatars.githubusercontent.com/u/873903?s=50" width="50" height="50">](https://github.com/jyknight)| @jyknight reported|
|-|-|
|Trac ID|trac#300|
|Type|enhancement|
|Created|2003-09-29 23:13:12Z|

<details><summary>Searchable metadata</summary>

```
trac-id__300 300
type__enhancement enhancement
reporter__jknight jknight
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1064877192000000 1064877192000000
changetime__1468910883618262 1468910883618262
version__ 
owner__ 
cc__radix cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jyknight's avatar" src="https://avatars.githubusercontent.com/u/873903?s=50" width="50" height="50">](https://github.com/jyknight)|@jyknight commented|
|-|-|

```
#!html
<pre>
http://roundup.sourceforge.net/doc-0.6/customizing.html seems to 
have some information that might be relevant to fixing this.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
depends on #245

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
It is now!

http://twistedmatrix.com/cgi-bin/mailman/listinfo/twisted-bugs
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset c81c502e401dd85725d45e79c0201b4e6e6e6a49
```
#!CommitTicketReference repository="" revision="c81c502e401dd85725d45e79c0201b4e6e6e6a49"
Merge pull request #300 from twisted/8211-rodrigc-inotify-py3

Author: rodrigc
Reviewer: Lukasa, adiroiban
Fixes: #8211

Port twisted.internet.inotify and twisted.python._inotify to Python 3
```
